### ISSUE TITLE
Create independent training and test sets

### ISSUE THREAD

**Issue by ThatOneGoat** [ThatOneGoat/sidewalk-cv-2021#10]
When trying to generate the dataset for our first official round of training, I ended up generating a large dataset of about 50k labels and then splitting that dataset randomly into an 80/10/10 split for train/val/test. However, this solution doesn't work well if we want to use the test set across multiple trainings/evaluations as the test set is randomly generated each time. 

Therefore, the best way to resolve this is to just create a smaller test set independent of the training set and adjust the code to load the train/test sets independently.
